setup.py

```diff
@@ -12,6 +12,8 @@
     "decorator",
     "memory-profiler",
     "parameterized",
+    "pybedtools",
```

Comment:
please add these dependencies to `docker/openproblems-python-extras/requirements.txt` instead and set this to be the associated docker image for your method

Reply:
do we need `memory-profiler` as a dependency? Don't we profile memory use automatically via the evaluation pipeline?